pkg/transportrequest/cts/create_test.go

```diff
@@ -0,0 +1,106 @@
+package cts
+
+import (
+	"bytes"
+	"errors"
+	"github.com/SAP/jenkins-library/pkg/mock"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestCreateTransportRequest(t *testing.T) {
+
+	defaultCreateAction := CreateAction{
+		Connection: Connection{
+			Endpoint: "https://example.org/cts",
+			User:     "me",
+			Password: "******",
+		},
+		TransportType:  "X",
+		TargetSystemID: "XYZ",
+		Description:    "Lorem ipsum",
+		CMOpts:         []string{"-Dx=y", "-Dabc=123"},
+	}
+
+	t.Run("straight forward", func(t *testing.T) {
+		examinee := defaultCreateAction
+		cmd := getExecMock()
+		cmd.StdoutReturn = map[string]string{"cmclient:*": "XXXK12345678"}
+		transportRequestId, err := examinee.Perform(cmd)
+
+		if assert.NoError(t, err) {
+			t.Run("assert the call", func(t *testing.T) {
+				assert.Equal(t, []mock.ExecCall{
+					mock.ExecCall{
```

Comment:
vscode complains: "redundant type from array, slice, or map composite literal".

you  can omit the `mock.ExecCall` literal.